src/dmd/mars.d

```diff
@@ -783,15 +783,16 @@ private int tryMain(size_t argc, const(char)** argv)
     if (global.errors)
         fatal();
 
-    backend_init();
-
     // Do semantic analysis
     foreach (m; modules)
     {
         if (global.params.verbose)
             message("semantic  %s", m.toChars());
         m.dsymbolSemantic(null);
     }
+
+    backend_init();
```

Comment:
The backend is dependent on whether `Throwable` is declared, so we have to move its initialization until after `dsymbolSemantic`.